test/unit/ssz_ex_test.exs

```diff
@@ -22,4 +22,27 @@ defmodule Unit.SSZExTest do
     assert_roundtrip(<<1>>, true, :bool)
     assert_roundtrip(<<0>>, false, :bool)
   end
+
+  test "serialize and deserialize list" do
+    assert_roundtrip(<<1, 1, 0>>, [true, true, false], {:list, :bool, 3})
```

Comment:
We should add a comment here clarifying that this schema shouldn't be used in practice. I was thinking this encoding was weirdly inefficient, but that's because this is a _boolean list_ instead of a _bitlist_.